src/extension.ts

```diff
@@ -57,11 +57,17 @@ async function gitBranch(repoDir: string) {
   });
 }
 
+/** 
+ * Default value for sourcegraph.url configuration.
+ */
+const DEFAULT_URL = "https://sourcegraph.com";
+
 function sourcegraphURL() {
-  // Support old "URL" key or new "url" key.
-  const url =
-    vscode.workspace.getConfiguration("sourcegraph").get<string>("url") ||
-    vscode.workspace.getConfiguration("sourcegraph").get<string>("URL");
+  let url = vscode.workspace.getConfiguration("sourcegraph").get<string>("url");
+  // Use remote.endpoint if it is configured and sourcegraph.url is not.
+  if (url === DEFAULT_URL) {
+    url = vscode.workspace.getConfiguration("remote").get<string>("endpoint") || DEFAULT_URL;
+  }
```

Comment:
Why not just do `remote.endpoint || sourcegraph.url` effectively? I don't like the `url === DEFAULT_URL` implementation here.

We don't have to advertise that we support `remote.endpoint` to VS Code users, and it saves us from having bugs where `DEFAULT_URL` here does not match what is in `package.json`.

Reply:
ie
```
  // Support "remote.endpoint" first (Sourcegraph Editor) and "sourcegraph.url" second (VS Code).
  const url =
    vscode.workspace.getConfiguration("remote").get<string>("endpoint") ||
    vscode.workspace.getConfiguration("sourcegraph").get<string>("url");
```